data/saas/dataset/power_reviews_dataset.yml

```
@@ -0,0 +1,15 @@
+dataset:
+  - fides_key: <instance_fides_key>
+    name: power_reviews
+    description: A sample dataset representing the PowerReviews connector for Fides
+    collections:
+      - name: privacy
+        fields:
+          - name: request_id
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: status
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
```

This can be empty since we don't have an access request
```suggestion
    collections: []
```